janitor/functions/get_dupes.py

```diff
@@ -16,20 +17,55 @@ def get_dupes(
 
     This method does not mutate the original DataFrame.
 
-    Functional usage syntax:
-
     ```python
     df = pd.DataFrame(...)
-    df = get_dupes(df)
+    janitor.get_dupes(df)
     ```
```

Comment:
@ethompsy could we remove this Markdown syntax block, please?

Reply:
Done!